content/uiux/concepts/bias-in-user-research/bias-in-user-research.md

```diff
@@ -0,0 +1,111 @@
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+A researcher runs a concept test to evaluate an early design idea. The participant may believe that the goal is to validate the idea so the company can move on to the development phase. This belief could influence them to try to “help” the researcher. They might provide positive feedback even if they don’t like the idea. </details> 
+
+### Framing Effect 
+
+The __framing effect__ occurs when findings are presented in a way that is disingenuous. Examples of the framing effect could be excluding certain data points, using statistical measures that are skewed, or strategically phrasing an insight to imply a certain idea or opinion. Researchers should present the data in a way that accurately represents respondents’ statements. 
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+Imagine that 20 survey respondents rated their overall approval of a certain product on a Likert scale of 1 to 5.  
+
+![Likert Scale Bar Chart that displays the data from the survey question: Approval from 1-5](media/framing-effect.png) 
+
+The researcher reports that the average approval rate is between 2 and 3, meaning that users are slightly dissatisfied. Considering that the mean is 2.45, this is technically correct. But it is a disingenuous finding &mdash; only 25% of the users actually rated the product between a 2 and a 3.  
+
+It would be more accurate to report that the majority of users are either extremely dissatisfied or extremely satisfied with the product. By reporting the finding that the approval rate is between 2 and 3, the team might miss the root of the issue that causes disparities in the satisfaction of certain groups of users. </details> 
+
+### Interview Bias 
+
+__Interview bias__ refers to a participant's perception of the interviewer. This may be due to a user's own biased perception of an interviewer's ethnicity, age, gender, or physical appearance &mdash; or even details like body language or facial expressions. Interview bias can be difficult to avoid, so it is just important to keep in mind when evaluating the data. 
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+Imagine that a team of four researchers conducts user interviews. Three of the researchers are 25 years old and one researcher is 50 years old. All interview participants are 20-25 years old.  
+
+When reviewing the data, the team noticed a clear trend: participants who talked to the younger researchers shared detailed responses that indicated honesty and openness. By contrast, most participants who spoke with the older researcher shared little detail, and their responses indicated an unwillingness to share openly. In this case, participants may have held a bias against the older researcher, which impacted how they approached the interview. </details> 
```

Comment:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

Imagine that a team of four researchers conducts user interviews. Three of the researchers are 25 years old and one researcher is 50 years old. All interview participants are 20-25 years old.  

When reviewing the data, the team noticed a clear trend: participants who talked to the younger researchers shared detailed responses that indicated honesty and openness. By contrast, most participants who spoke with the older researcher shared little detail, and their responses indicated an unwillingness to share openly. In this case, participants may have held a bias against the older researcher, which impacted how they approached the interview. </details> 
```
```